lib/iris/tests/results/abf/load.cml

```diff
@@ -34,6 +34,6 @@
       </coord>
     </coords>
     <cellMethods/>
-    <data checksum="0xbf5ea7c7" dtype="uint8" fill_value="255" mask_checksum="0x90665a57" shape="(2160, 4320)"/>
+    <data checksum="0xbf5ea7c7" dtype="uint8" mask_checksum="0x90665a57" shape="(2160, 4320)"/>
   </cube>
 </cubes>
```

Comment:
This is good proof that #2548 was fixed correctly.

When the issue that PR was discovered this test was creating a fill_value of 63 which is no longer the case.